backend/vehicles/src/trailer-types/dto/request/create-trailer-type.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateTrailerTypeDto {
+  @AutoMap()
+  @ApiProperty({ example: 'BOOSTER', description: 'The Trailer Type Code' })
+  typeCode: string;
+
+  @AutoMap()
+  @ApiProperty({ example: 'Boosters', description: 'Trailer Type' })
+  type: string;
```

Comment:
needs a better description IMO. Otherwise, it can be confusing to distinguish between type and typeCode.

Reply:
Resolved